test/php/library/Director/Web/Form/Element/DictionaryTest.php

```diff
@@ -32,53 +32,6 @@ public function testValidDictionary() {
         $this->assertFalse($this->dictionaryInstance->hasErrors());
     }
 
-    public function testDictionaryWithMissingKey() {
```

Comment:
We used to test this because we threw an error when a key was missing. Now, it's okay, if you delete an entire key, it will get replaced by its default value. Since "all this" is a temporary solution, I think it's good as it is.